velox/docs/functions/string.rst

```diff
@@ -127,6 +127,18 @@ String Functions
 
     Converts ``string`` to uppercase.
 
+.. function:: hamming_distance(string1, string2) -> bigint
```

Comment:
Functions in the docs should appear in alphabetical order. Would you move the new functions around to preserve that order?